jupyterlab_gcpscheduler/src/components/jobs_panel_widget.tsx

```diff
@@ -97,9 +97,20 @@ const localStyles = stylesheet({
   },
 });
 
+const StyledTabs = withStyles({
+  indicator: {
+    backgroundColor: 'rgba(0,0,0,0)',
```

Comment:
Should we try to use one of the JupyterLab variable colors to protect us against having to address this if/when we want to support a dark theme?

Reply:
Ah this is me overriding the default indicator (which had a responsiveness issue) and setting it to transparent (the 0 alpha value)

Reply:
Gotcha!